VirtusLab-Open-Source/strapi-plugin-comments
# feat: comments pre-moderation

## Ticket

Issue #45 

## Summary

What does this PR do/solve? 

This PR adds pre moderation of the comments. Ability to mark as pending, displaying said status and deciding about approval status in admins panel.

## Test Plan

- Add comment with body containing `approvalStatus` === `PENDING`
- Enter admin's panel
- New comment should be marked as pending
- Approve comment
- Reject comment
- After that when querying one can filter by `approvalStatus`
